controllers/reviews/getReviewsByDishId/getReviewsByDishId.js

```diff
@@ -0,0 +1,44 @@
+import mongoose from 'mongoose';
+import { Review } from '../../../models/review/index.js';
+
+const ObjectId = mongoose.Types.ObjectId;
+
+export const getReviewsByDishId = (app) => {
+  app.get('/reviews/by-dish/:dishId', async (req, res, next) => {
+    try {
+      const { dishId } = req.params;
+      const reviews = await Review.aggregate([
+        {
+          $lookup: {
```

Comment:
you will populate all the reviews in the DB with dish data here
are you sure we don't need to `$match` reviews before doing `$lookup`?

Reply:
Sorry. I work on two computers. I created it at home on my own laptop. I haven't finished this part yet. I copied and pasted code from another file getReviewsByChefId and didn't change it because I plan to continue working on it using my work laptop in this branch. I pulled it only to have the opportunity to continue working.
Yes. I'm going doing it as you told, because dishId on top level.
Thank you. In future I will comment that I don't finished  task yet.